pkg/arvo/app/file-server.hoon

```diff
@@ -188,8 +188,11 @@
     ?:  ?=([%'~landscape' %js %session ~] site.req-line)
       %+  require-authorization-simple:app
         inbound-request
-      %-  js-response:gen
-      (as-octt:mimes:html "window.ship = '{+:(scow %p our.bowl)}';")
+      %.  %-  as-octs:mimes:html
+          :((cury cat 3) 'window.ship = "' (rsh 3 (scot %p our.bowl)) '";')
```

Comment:
`(rap 3 'window.ship = "' (rsh 3 (scot %p our.bowl)) '";' ~)` is preferable, as it's a single jetted call (and single allocation). (This specific case might be too small to matter, but it's worth establishing the habit.)